src/services/textService.js

```diff
@@ -0,0 +1,17 @@
+export default class TextService {
+  _data = {
```

Comment:
what do you think about supporting languages other than english?

Reply:
yep that's why I put text into separate TextService. We could modify it to return different text depending on user's language later on.